docker-compose.yml

```diff
@@ -33,8 +33,9 @@ services:
       context: .
       dockerfile: Dockerfile
     environment:
-      EVENTGATEWAY_KAFKA_PROXY_BROKERS_MAPPING: 'localhost:9092,proxy:28002'
-      EVENTGATEWAY_KAFKA_PROXY_BROKERS_DIAL_MAPPING: 'localhost:9092,broker:9092'
+#      EVENTGATEWAY_KAFKA_PROXY_BROKERS_MAPPING: 'localhost:9092,proxy:28002'
```

Comment:
Leaving it commented in purpose as both mechanisms are valid (explicit config VS AsyncAPI Doc)